test/integration/features/cmd-openshift.feature

```diff
@@ -31,6 +31,7 @@ cluster in VM provided by Minishift.
         On restart of OpenShift cluster containers are terminated, which sets FinishedAt to actual time. This value persist
         after next start of container.
      Given stdout of command "minishift ssh -- "docker inspect --format={{.State.FinishedAt}} origin"" is equal to "0001-01-01T00:00:00Z"
+       And user waits "10" seconds #TO BE REMOVED with OpenShift v3.9. OpenShift v3.7 needs additional time after startup to get containers ready due to #2169.
```

Comment:
@agajdosi Why the sleep is before the restart? I thought we need sleep after restart?

Reply:
@LalatenduMohanty I was suspecting the same but later I have found out the restart so soon after startup causes troubles. Issue: https://github.com/minishift/minishift/issues/2169

Reply:
Also we have moved to v3.9.0 recently. So we should try running it without sleep.